contracts/test/MessageTester.sol

```diff
@@ -5,12 +5,25 @@ import {FuelMessagesEnabled} from "../messaging/FuelMessagesEnabled.sol";
 import {IFuelMessagePortal} from "../messaging/IFuelMessagePortal.sol";
 
 /// @notice This contract allows for testing message receiving.
-contract MessageSendingContract is FuelMessagesEnabled {
+contract MessageTester is FuelMessagesEnabled {
+    bytes32 internal constant TRUSTED_SENDER = 0xf40001353a6b162f0ff9d59cae46ed49355aa4c424e3f79f6d84352f85715576;
+
+    /// @notice Storage to hold and test incoming data
+    uint256 public data1;
+    uint256 public data2;
+
     /// @notice Constructor.
     /// @param fuelMessagePortal The IFuelMessagePortal contract
-    // solhint-disable-next-line no-empty-blocks
-    constructor(IFuelMessagePortal fuelMessagePortal) FuelMessagesEnabled(fuelMessagePortal) {
-        //nothing to do
+    constructor(IFuelMessagePortal fuelMessagePortal) {
+        s_fuelMessagePortal = fuelMessagePortal;
+    }
+
+    /// @notice Message receiving function.
+    /// @param d1 Test param 1
+    /// @param d2 Test param 2
```

Comment:
They're just random parameters for the sake of having parameters to test with. I assume since the whole contract is called `MessageTester` it should be obvious the lack of documentation means they don't have any more sophisticated purpose other than to just exist for the tests.